cfme/configure/settings.py

```diff
@@ -367,5 +375,9 @@ def get_default_view(cls, button_group_name):
 class DefaultViewAll(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'MySettings')
 
+    def am_i_here(self):
+        if match_location(title='Configuration'):
```

Comment:
**Required**
Add `controller='configuration'` param to `match_location` since 'Configuration' title actually matches a few pages.